release/platform/platform.go

```diff
@@ -483,31 +475,42 @@ var platforms = []Platform{
 		BuildTags: defaultBuildTags,
 	},
 	{
-		Name:      "rhel82",
+		Name:      "rhel83",
+		Arch:      ArchS390x,
+		OS:        OSLinux,
+		Pkg:       PkgRPM,
+		Repos:     []Repo{RepoEnterprise},
+		BuildTags: defaultBuildTags,
+	},
```

Comment:
You said you removed RHEL 8.3 but it's still listed here.

Reply:
Sorry, I should correct my comment. I removed RHEL8.0, 8.2, and 9.0 variants, but not RHEL 8.3 because it's `ArchS390x`.